### PR TITLE
Add support for vGPU build

### PR DESCRIPTION
**Pull request by andr2000**
Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>
Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>

### CODE REVIEW

File: recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/init.d/rc.pvr [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+############################################################################ ###
+# File rc.pvr
+# Copyright Copyright (C) 2013-2016 Renesas Electronics Corporation
+# License Strictly Confidential.
+#### ###########################################################################
+
+# Auto-generated for r8a7796_linux from rogueddk_1.7@4495854
```
Comment (lorc) [inline recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/init.d/rc.pvr]: If it is autogenerated, why you are adding it?

File: recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/udev/rules.d/72-pvr-seat.rules [hunk 2]
```diff
@@ -0,0 +1,7 @@
+############################################################################ ###
+# File 72-pvr-services.rules
+# Copyright Copyright (C) 2013-2016 Renesas Electronics Corporation
+# License Strictly Confidential.
+#### ###########################################################################
+# Rules for PowerVR device nodes
+SUBSYSTEM=="drm", DEVPATH=="/devices/platform/soc/fd000000.gsx/drm/*", TAG+="seat", TAG+="master-of-seat", ENV{ID_AUTOSEAT}="1", ENV{ID_SEAT}="seat1"
```
Comment (lorc) [inline recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/udev/rules.d/72-pvr-seat.rules]: Will this work on all Renesas platforms (and under virtualization?)

File: recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/init.d/rc.pvr (hunk 1 above)
Comment (andr2000) [reply recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/init.d/rc.pvr]: although it is autogenerated it is not part of the git repo we have and was originally distributed as part of the binary package

File: recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/udev/rules.d/72-pvr-seat.rules (hunk 2 above)
Comment (andr2000) [reply recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/udev/rules.d/72-pvr-seat.rules]: it does at the moment if run as Dom0. After we move AGL to real DomD we will have to change this. ATM it is not clear what will be the right path...

**Review by lorc — APPROVED** [review]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@erpam.com>`

File: recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/init.d/rc.pvr (hunk 1 above)
Comment (aanisov) [reply recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/init.d/rc.pvr]: Isn't it generated during DDK build?

Comment (andr2000) [reply recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/init.d/rc.pvr]: I was, wrong, it is generated - will do

File: recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/udev/rules.d/72-pvr-seat.rules (hunk 2 above)
Comment (andr2000) [reply recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/udev/rules.d/72-pvr-seat.rules]: moved to the version supplied during the build/install by the PVR UM itself

**Comment by andr2000** [comment]
all fixed

**Review by aanisov — APPROVED** [review]

**Pull request merged**